app/models/pipeline_run.rb

```diff
@@ -108,9 +110,30 @@ class PipelineRun < ApplicationRecord
   STATUS_CHECKED = 'CHECKED'.freeze
   STATUS_FAILED = 'FAILED'.freeze
   STATUS_RUNNING = 'RUNNING'.freeze
-  STATUS_RUNNABLE = 'RUNNABLE'.freeze
+  STATUS_RUNNABLE = 'RUNNABLE'.freeze # TODO: (gdingle): not used anywhere?
```

Comment:
`STATUS_RUNNABLE` is not referred to anywhere in idseq-web and it does not appear in the prod DB so it's not clear how it supposed to be used